Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -438,6 +438,20 @@ class BrowserCoordinator: BaseCoordinator,
         return bottomSheetCoordinator
     }
 
+    func showQRCode() {
+        let coordinator = makeQRCodeCoordinator()
+        coordinator.showQRCode(delegate: browserViewController)
+    }
+
+    private func makeQRCodeCoordinator() -> QRCodeCoordinator {
```

Comment:
Is it really necessary to have a `makeQRCodeCoordinator` method here? Could we just create the coordinator in `showQRCode`?

Reply:
Yes you are right, i will move it to the show qr code method